runtime.go

```diff
@@ -50,8 +50,8 @@ func NewRuntimeStats(scope Scope) StatGenerator {
 
 		nextGC:       scope.NewGauge("nextGC"),
 		lastGC:       scope.NewGauge("lastGC"),
-		pauseTotalNs: scope.NewCounter("pauseTotalNs"),
-		numGC:        scope.NewCounter("numGC"),
+		pauseTotalNs: scope.NewGauge("pauseTotalNs"),
```

Comment:
https://cs.opensource.google/go/go/+/refs/tags/go1.24.3:src/runtime/mstats.go;l=258-263

>PauseTotalNs is the cumulative nanoseconds in GC
	// stop-the-world pauses since the program started
	
so pauseTotalNs is not a gauge, it is actually a counter

Reply:
yeah I think we were looking at one of the broken charts wrong and the reason it was broken I gave more context in the thread https://lyft.slack.com/archives/C0KLXB3KN/p1747250876491629 and updated the doc